Tests/ExtractDataTests.m

```diff
@@ -111,12 +111,12 @@ - (void)testExtractData_NoPassword
 
 - (void)testExtractData_IncorrectPassword
 {
-    NSString *testArchiveName = @"Test Archive (Bad Password).rar";
+    NSString *testArchiveName = @"Test Archive (RAR5, Password).rar";
     NSURL *testArchiveURL = self.testFileURLs[testArchiveName];
-    URKArchive *archive = [[URKArchive alloc] initWithURL:testArchiveURL password: @"password2" error: nil]; //the correct password is 'password'
+    URKArchive *archive = [[URKArchive alloc] initWithURL:testArchiveURL password: @"wrong password" error: nil];
     
     NSError *error = nil;
-    NSData *data = [archive extractDataFromFile:@"test.txt" error:&error];
+    NSData *data = [archive extractDataFromFile:@"wtf.txt" error:&error];
```

Comment:
Why make this an invalid value (`wtf`)?

Reply:
wtf is actually a valid file inside rar5 file. It makes it invalid because the test doesn't provide a valid password

Reply:
Haha I totally forgot that. Bad choice on my part. Looks good!